### DISCUSSION TITLE
Clicking on a door or hatch (to open it) while placing a wire shouldn't place the wire

### DISCUSSION THREAD

**Discussion by UserC2** [FakeFishGames/Barotrauma#14330] [Ideas and suggestions]
This would make it much easier to run wires through different rooms.

**Comment by kaedepon** [comment]
I think it's cool that placing a wire like any wiring components what requires holding the right mouse button and left click.

**Reply by UserC2** [reply]
That would make it much easier to control wire placement.

**Comment by gerakis** [comment]
You can always right click to go back by one wire node. Its easy to undo a misclick.
